config/routes.rb

```diff
@@ -3,7 +3,11 @@
 
   devise_for :users
 
-  resources :papers
+  resources :papers do
+    collection do
+      get 'pubmed_search'
```

Comment:
Here are my thoughts on this route right now:

Do we want a pubmed specific route? When we have to search our own papers, is that something that we'll want on the same route as searching for pubmed papers?

Just a thought.